prow/plugins/trigger/ic.go

```diff
@@ -26,7 +26,7 @@ import (
 	"k8s.io/test-infra/prow/plugins"
 )
 
-var okToTest = regexp.MustCompile(`(?m)^(@k8s-bot )?ok to test\r?$`)
+var okToTest = regexp.MustCompile(`/ok-to-test`)
```

Comment:
This should a) start recommending `/ok-to-test` but continue to accept `@k8s-bot ok to test` for awhile.

Also please add unit tests to validate that both `@k8s-bot ok to test` and `/ok-to-test` work